src/overview/reducer.js

```diff
@@ -264,6 +268,32 @@ const setFilterPopup = (state, name) => {
     return { ...state, filterPopup }
 }
 
+const setShowTooltip = (state, { isShowTooltip }) => {
+    return {
+        ...state,
+        showTooltip: isShowTooltip || !state.showTooltip,
+    }
+}
+
+const setTooltipIndex = state => {
```

Comment:
This is way too complex. Wouldn't just incrementing the `tooltipIndex` suffice here? (perform `% tooltips.length` in the selector to keep within bounds). Probably don't need these states anyway when you change the single active `tooltip` state - keep in mind